### PR TITLE
Use individual component sass

### PR DESCRIPTION
**Pull request by andysellick**
Update feedback to import the sass for only the components it is using, instead of all components, as [documented here](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/install-and-use.md#import-sass-for-individual-components).

## Filesize comparison

Before:

Screenshot 2020-04-20 at 17 06 55

Total **884 KB**

After:

Screenshot 2020-04-23 at 08 43 55

Total **254 KB**

## Additional changes

Many of the gem components in this application had been implemented using markup copy and paste instead of including them directly from the gem, specifically on the /contact/govuk/ page. This has now been updated.

Screenshot 2020-04-23 at 08 46 32

Links to test (may not be complete):

- https://www.gov.uk/contact/
- https://www.gov.uk/contact/govuk
- https://www.gov.uk/contact/govuk/thankyou
- https://www.gov.uk/contact/govuk/anonymous-feedback/thankyou

Trello card: https://trello.com/c/GWttlBxs/228-upgrade-applications-to-use-individual-component-sass

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@import "govuk_publishing_components/components/hint";
+@import "govuk_publishing_components/components/input";
+@import "govuk_publishing_components/components/label";
+@import "govuk_publishing_components/components/radio";
+@import "govuk_publishing_components/components/textarea";
+@import "govuk_publishing_components/components/title";
+@import "govuk_publishing_components/components/warning-text";
+
+.app-link-as-heading {
+    @include govuk-font($size:24, $weight: bold, $line-height: 1.8);
```
Comment (alex-ju) [inline app/assets/stylesheets/application.scss:24]:
```suggestion
  @include govuk-font($size:24, $weight: bold, $line-height: 1.8);
```

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@charset "UTF-8";
```
Comment (alex-ju) [inline app/assets/stylesheets/application.scss:1]: how come we need this explicitly?

File: README.md [hunk 3]
```diff
@@ -19,6 +19,15 @@ submitted via an anonymous contact form in the [feedback app][feedback].
 via a form that will require you to identify yourself. This data is sent directly
 to the `support` app.
 
+## Pages rendered by the application
```
Comment (alex-ju) [inline README.md:22]: 🙌

File: app/views/contact/govuk/new.html.erb [hunk 4]
```diff
@@ -9,94 +9,129 @@
...
-      # which we do not want to display in the error summary
-      # so we filter it out
-      items: @errors.keys.reject { |key| key == :giraffe }.map do |key| 
+      <%= render "govuk_publishing_components/components/error_summary", {
+        id: "error-summary",
+        title: "Please check the form",
+        # app adds a spam protection giraffe error,
+        # which we do not want to display in the error summary
+        # so we filter it out
+        items: @errors.keys.reject { |key| key == :giraffe }.map do |key|
```
Comment (alex-ju) [inline app/views/contact/govuk/new.html.erb:18]: 🦒

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (andysellick) [reply app/assets/stylesheets/application.scss:1]: I've no idea. I think this line was already in this file when I renamed it. I'll try taking it out.

Comment (andysellick) [reply app/assets/stylesheets/application.scss]: Have removed. Seems to be okay.

**Pull request merged**